src/morphydefs.h

```diff
@@ -28,7 +28,7 @@ typedef double Mflt;
 typedef unsigned int MPLstate;
 
 #define NA              ((MPLstate)0b1)
```

Comment:
Just a quick comment, not that you guys care, but typically I like to prefix things like this, it makes the code more greppable. Try and grep the codebase for NA and see what you get... If it was something like MORPHY_NA, or MFL_ISMISSING etc... then it doesn't get lost so easily.

Reply:
Funny, I just read the comment on why the MISSING define was changed to ISMISSING... that is yet another reason to have prefixes.